api/handlers/service_binding_test.go

```diff
@@ -568,4 +574,44 @@ var _ = Describe("ServiceBinding", func() {
 			})
 		})
 	})
+
+	Describe("GET /v3/service_credential_bindings/{guid}/details", func() {
+		BeforeEach(func() {
+			requestMethod = http.MethodGet
+			requestPath = "/v3/service_credential_bindings/service-binding-guid/details"
+			requestBody = ""
+		})
+
+		It("returns the service binding details", func() {
+			Expect(rr).To(HaveHTTPStatus(http.StatusOK))
+			Expect(rr).To(HaveHTTPHeaderWithValue("Content-Type", "application/json"))
+			Expect(rr).To(HaveHTTPBody(SatisfyAll(
+				MatchJSONPath("$.credentials.connection", "mydb://user@password:example.com"),
+				MatchJSONPath("$.syslog_drain_url", "http://syslog.example.com/drain"),
```

Comment:
With regards to `syslog_drain_url` and `volume_mounts`, we would better not return anything. Furnthermore these values are not stubbed in the before each, so this test would probably fail.